### ISSUE TITLE
Feature format: HTK, Kaldi, or custom?

### ISSUE THREAD

**Issue by arlofaria** [remeeting/mrp-go#4]
Currently we use a custom 4-byte header.  It might be nice to adopt HTK's 12-byte header ... always set the parm kind to `USER=9`, implement the bit flags for encoding deltas (`_A` and `_D`) and possibly also compression (`_C`).

**Comment by mkjois** [comment]
I believe the features are HTK-compliant. Why not set the parm kind to MFCC (I think it was 6)?

**Comment by arlofaria** [comment]
Yes, I believe the features are nearly identical to those produced by HTK.  We could set the `ParmKind` to `MFCC`.  But for them to be truly HTK-compliant as a file format, we need to create the 12-byte header.  When produced in streaming mode, I guess we could set the length/duration to -1.

However, it might make more sense to compute Kaldi-compatible features...
